code/controllers/master_controller.dm

```diff
@@ -318,13 +318,8 @@ datum/controller/game_controller/proc/process()
 
 /datum/controller/game_controller/proc/processPowernets()
 	last_thing_processed = /datum/powernet
-
-	for (var/datum/powernet/Powernet in powernets)
-		if (Powernet)
-			Powernet.reset()
-			continue
-
-		powernets -= Powernet
+	for(var/datum/powernet/Powernet in powernets)
+		Powernet.reset()

```

Comment:
correct me if I'm wrong (just double checking here), but this never runs, correct (because we don't use the MC anymore).